js/backend.js

```diff
@@ -35,9 +33,10 @@ $('#trainNew').on('click', function () {
             // on close get file path
```

Comment:
I don't think it's you, I just have no idea why this worked fine before, but i think the issue is line 29: 
change  `dialog.showOpenDialog({`  to   `dialog.showOpenDialog(remote.getCurrentWindow(), {` 
maybe fixes.  This forces the dialog box to attach to the window, instead of floating, so it can't disappear

Reply:
oh wait, json upload is broken somewhere